apps/vc-api/src/vc-api/exchanges/dtos/submission-review.dto.ts

```diff
@@ -18,24 +18,27 @@
 import { Type } from 'class-transformer';
 import { IsEnum, IsOptional, ValidateNested } from 'class-validator';
 import { VerifiablePresentationDto } from '../../credentials/dtos/verifiable-presentation.dto';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 
 export enum ReviewResult {
   approved = 'approved',
   rejected = 'rejected'
 }
 
 export class SubmissionReviewDto {
-  /**
-   * The judgement made by the reviewer
-   */
   @IsEnum(ReviewResult)
+  @ApiProperty({
+    description: 'The judgement made by the reviewer',
+    enum: ReviewResult
+    // enumName: 'ReviewResult'
```

Comment:
A note to remove commented code
```suggestion
```

Reply:
I will uncomment this later. Now, it is left like this for easier comparison with the baseline of the openapi.json. A good practice is to name enums.